docs/en/ingest-management/commands.asciidoc

```diff
@@ -36,19 +36,29 @@ invoking the wrong binary.
 [[elastic-agent-diagnostics-command]]
 == elastic-agent diagnostics
 
-Gather diagnostics information from the {agent} and applications it's running.
+Gather diagnostics information from the {agent} and component/unit it's running.
 
-If no options are specified, this command displays version numbers and application metadata.
+If no options are specified, this command displays version numbers and component/unit metadata.
```

Comment:
This is not the case any more. There is no `collect` subcommand anymore. It just `elastic-agent diagnostics` and it will collect all the files.